modules/scylladb/src/test/java/org/testcontainers/scylladb/ScyllaDBContainerTest.java

```diff
@@ -0,0 +1,199 @@
+            ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE)
+                .withConfigurationOverride("scylla-test-ssl")
+            // }
+        ) {
+            // sslContext {
+            ScyllaDBContainer sslScylladb = scylladb.withSsl(
+                MountableFile.forClasspathResource("scylla-test-ssl/keys/scylla.cer.pem"),
+                MountableFile.forClasspathResource("scylla-test-ssl/keys/scylla.key.pem"),
+                MountableFile.forClasspathResource("scylla-test-ssl/keys/scylla.truststore")
+            );
+
+            String testResourcesDir = getClass().getClassLoader().getResource("scylla-test-ssl/keys/").getPath();
+
+            KeyStore keyStore = KeyStore.getInstance("PKCS12");
+            keyStore.load(Files.newInputStream(Paths.get(testResourcesDir + "scylla.keystore")), "scylla".toCharArray());
+
+            KeyStore trustStore = KeyStore.getInstance("PKCS12");
+            trustStore.load(
+                Files.newInputStream(Paths.get(testResourcesDir + "scylla.truststore")),
+                "scylla".toCharArray()
+            );
+
+            KeyManagerFactory keyManagerFactory = KeyManagerFactory.getInstance(
+                KeyManagerFactory.getDefaultAlgorithm()
+            );
+            keyManagerFactory.init(keyStore, "scylla".toCharArray());
+
+            TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(
+                TrustManagerFactory.getDefaultAlgorithm()
+            );
+            trustManagerFactory.init(trustStore);
+
+            SSLContext sslContext = SSLContext.getInstance("TLS");
+            sslContext.init(keyManagerFactory.getKeyManagers(), trustManagerFactory.getTrustManagers(), null);
+            // }
+
+            sslScylladb.start();
+
+            CqlSession session = CqlSession
+                .builder()
+                .addContactPoint(sslScylladb.getContactPoint())
+                .withLocalDatacenter("datacenter1")
+                .withSslContext(sslContext)
+                .build();
+            ResultSet resultSet = session.execute(BASIC_QUERY);
+            assertThat(resultSet.wasApplied()).isTrue();
+            assertThat(resultSet.one().getString(0)).isNotNull();
+            assertThat(session.getMetadata().getNodes().values()).hasSize(1);
+        }
+    }
+
+    @Test
+    public void testSimpleSslCqlsh() throws IllegalStateException, InterruptedException, IOException {
+        // sslConfiguration {
+        // }
+        try (
+            ScyllaDBContainer scylladb = new ScyllaDBContainer(SCYLLADB_IMAGE)
+                .withConfigurationOverride("scylla-test-ssl")
+                .withSsl(
+                    MountableFile.forClasspathResource("scylla-test-ssl/keys/scylla.cer.pem"),
+                    MountableFile.forClasspathResource("scylla-test-ssl/keys/scylla.key.pem"),
+                    MountableFile.forClasspathResource("scylla-test-ssl/keys/scylla.truststore")
```

Comment:
pem files and truststore are copied twice. One as part of `withConfigurationOverride` and other `withSsl`. I suggest to move the certificates from `src/test/resources/scylla-test-ssl/keys` under `src/test/resources`

Reply:
thanks, fixed it